main.js

```diff
@@ -198,12 +209,53 @@ function showView(view) {
 }
 
 function renderCartCount(cart) {
-  var cartCount = createElement('div', { style: 'height: 30px' }, [
-    createElement('div', { class: 'p-5 float-right' }, ['Cart (' + cart.length + ')'])
+  var cartCount = createElement('div', null, [
+    createElement('div', { id: 'cart', class: 'p-2 float-right' }, ['Cart (' + cart.length + ')'])
   ])
   return cartCount
 }
 
+function renderCartItem(cartItem) {
+  var cartElement = createElement('div', { class: 'd-flex justify-content-center m-auto', style: 'width: 30rem' }, [
+    createElement('div', { class: 'row border', style: 'height: 9.5rem' }, [
+      createElement('div', { class: 'col-sm-4 ' }, [
+        createElement('img', { class: 'img-thumbnail border-0', src: cartItem.imageUrl }, [])
+      ]),
+      createElement('div', null, [
+        createElement('div', { class: 'card-body' }, [
+          createElement('h5', null, [cartItem.name]),
+          createElement('h6', null, ['Brand: ' + cartItem.brand]),
+          createElement('p', null, ['$' + cartItem.price])
+        ])
+      ])
+    ])
+  ])
+  return cartElement
+}
+
+function renderCartPage(cart) {
```

Comment:
This function would be a lot cleaner if you didn't calculate the price and render the cart items in the same loop.